app/controllers/AccountManagement.scala

```diff
@@ -118,15 +118,22 @@ object AccountManagement extends Controller with LazyLogging {
       pendingHolidays = pendingHolidayRefunds(newHolidays)
```

Comment:
This wasn't your code but there's a call to getHolidays in this file that returnsString \/ Seq[Holiday] where the string is going to be some Zuora JSON we failed to deserialise, and it'll be printed out in BadRequest unless we leftMap it to something like Internal Error first